Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/entitlements_page.tsx

```diff
@@ -1,6 +1,9 @@
 import * as Beam from 'beam';
 import * as Nexus from 'nexus';
 import * as React from 'react';
+import {DisplaySize, HBoxLayout, HLine, Padding, VBoxLayout} from '../../..';
```

Comment:
90% of Typescript code uses ```import { x }``` instead of ```import {x}```, so we're going to join the masses and surround braces with spaces.